akka-bbb-fsesl/build.sbt

```diff
@@ -28,8 +28,8 @@ val compileSettings = Seq(
 )
 
 resolvers ++= Seq(
-  "spray repo" at "http://repo.spray.io/",
-  "blindside-repos" at "http://blindside.googlecode.com/svn/repository/"
+  ("spray repo" at "http://repo.spray.io/").withAllowInsecureProtocol(true),
```

Comment:
- this is http only
- this only includes an "old" version of spray
- spray.io is not maintained any more for more than half a decade. Please migrate to akka http or similar. Also see https://doc.akka.io/docs/akka-http/current/migration-guide/migration-from-spray.html?language=scala